tools/ci/shard_packages

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env zx
+
+await runAtGitRoot()
+
+$.verbose = false
+const numOfShards = Number(process.env.SHARDS) || 5
+const testType = process.env.TEST_TYPE
+// parallel array mapping
+const validTestTypes = ['go_core_tests', 'go_core_race_tests']
+const testTypeToList = ['./...', './core/...']
```

Comment:
Oh I think we can simplify by race testing `./...`. I don't think we were limiting to just `core/` by design - probably just oversight.